.ci/install-yq.sh

```diff
@@ -51,6 +51,11 @@ function install_yq() {
 
 	mkdir -p "${GOPATH}/bin"
 
+	# Check curl
+	if ! command -v "curl" >/dev/null; then
+		die "Please install curl"
```

Comment:
It may confuse users because he/she don't want to install curl originally. Is the following one better:

> Failed to install yq: no curl installed